### PR TITLE
Moved closing of kryoOutput to finally block

### PR DESCRIPTION
**Pull request by denis256**
Noticed that in io.paperdb.DbStoragePlainFile#writeTableFile, in case of exception kryoOutput will remain open, which will lead to leaking of opened file descriptors...
Moved stream closing code to finally block

### CODE REVIEW

**Comment by pilgr** [comment]
@denis256 thanks! Please make changes to the development branch.

File: paperdb/src/main/java/io/paperdb/DbStoragePlainFile.java [hunk 1]
```diff
@@ -278,7 +278,7 @@ private File getOriginalFile(String key) {
                     "Backed up table will be used on next read attempt", e);
         } finally {
             if (kryoOutput != null) {
-                kryoOutput.close();  //also close file stream
+                kryoOutput.close();  // closing opened kryo output
```
Comment (pilgr) [inline paperdb/src/main/java/io/paperdb/DbStoragePlainFile.java]: it's essential to note that closing kryoOutput also closes corresponding file stream, please leave the comment.

Comment (denis256) [reply paperdb/src/main/java/io/paperdb/DbStoragePlainFile.java]: Comment updated

**Comment by denis256** [comment]
Hi, I see destination branch was changed to *development*, I need to do more things for promoting chnages into *development* ?

**Comment by pilgr** [comment]
Yep, please add your 3 commits on top of the development branch. I mean you can add your changes to the development branch in your fork (just cherry pick them from master in your local fork). Then either create new PR or change this one to have `from` as  `unidev-android:development`. Thank you!

P.S> I need to add this info to the contribution section of readme 🤔

**Pull request closed without merging**